data_hub_api/docmaps/v2/codecs/docmaps.py

```diff
@@ -310,17 +310,15 @@ def get_docmap_item_for_query_result_item(query_result_item: ApiInput) -> Docmap
     manuscript_first_version = query_result_item['manuscript_versions'][0]
     qc_complete_timestamp_str = manuscript_first_version['qc_complete_timestamp'].isoformat()
     id_query_param = {'manuscript_id': query_result_item['manuscript_id']}
-    publisher_json = query_result_item['publisher_json']
+    publisher_json = ast.literal_eval(query_result_item['publisher_json'])
```

Comment:
Why don't we just use `json.loads`? Even though `literal_eval` seems relatively safe in that it can't execute malicious code, it still seems the wrong tool.

And since we are currently not planning to add other publishers, we could potentially even just hard-code the provider.

(I think a bit confusion comes from `PARSE_JSON` seemingly still returning a string encoded JSON in the end)